client/src/components/product-card/index.tsx

```diff
@@ -15,9 +15,9 @@ export interface ProductCardProps {
 
 const ProductCard = ({ product }: ProductCardProps) => {
   return (
-    <div className='aspect-[250/320] size-full max-w-64 overflow-hidden rounded-xl bg-card'>
+    <div className='aspect-[170/240] size-full max-w-64 overflow-hidden rounded-xl bg-card shadow-cardbox md:aspect-[250/320]'>
       <div className='relative h-2/5 w-full'>
-        <Image src={product.image} alt={product.name} fill />
+        <Image src={product.photo as string} alt={product.name} fill />
```

Comment:
เดะ undefined -> "undefined"

```suggestion
        <Image src={product.photo ?? ''} alt={product.name} fill />
```